Tests/HtcMock/Client/src/SessionClient.cs

```diff
@@ -136,8 +136,7 @@ public IEnumerable<string> SubmitTasksWithDependencies(IEnumerable<Tuple<byte[],
 
     foreach (var (payload, dependencies) in payloadsWithDependencies)
     {
-      var taskId = Guid.NewGuid()
-                       .ToString();
+      var taskId = "root";
```

Comment:
Won't this change imply mutiple subtasks with identical ids in one session?

Reply:
Yes, that is intended to produce more reliably the error we try to recover from.

Reply:
In the future, we want that the task id generated by the submitter so it will fix this issue